aten/src/ATen/native/native_functions.yaml

```diff
@@ -5524,14 +5524,14 @@
 - func: eig.e(Tensor self, bool eigenvectors=False, *, Tensor(a!) e, Tensor(b!) v) -> (Tensor(a!) eigenvalues, Tensor(b!) eigenvectors)
   dispatch:
     CPU: legacy::cpu::_th_eig_out
-    CUDA: legacy::cuda::_th_eig_out
+    CUDA: eig_cuda_out
```

Comment:
Are these custom CUDA dispatches necessary?

Reply:
well, we have two different implementations, one for CPU and one for CUDA. Is there another way to dispatch to these?